doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -218,6 +218,11 @@ nRF Machine Learning (Edge Impulse)
 
 |no_changes_yet_note|
 
+Protocols serialization
+-----------------------
+
+* Added the protocols serialization server application.
```

Comment:
```suggestion
* Added the Protocols serialization server application.
```
Sorry - removing that capitalization was unintended - since it's a name we should keep it capitalized

Reply:
no problem should be fine now.